src/send-query.js

```diff
@@ -0,0 +1,12 @@
+import ajax from './ajax';
+
+export default function sendQuery(query, config) {
+  const url = `https://${config.domain}/api/graph`;
+  const headers = {
+    authorization: `Basic ${btoa(config.apiKey)}`,
```

Comment:
hits my head ... it's imported in `shopify.js` ... sometimes, i just forget things.